### DISCUSSION TITLE
When can Get the latest  EJBCA Community  V9.0 on github?

### DISCUSSION THREAD

**Discussion by vanlee90** [Keyfactor/ejbca-ce#687] [General]
EJBCA Community V9.0 has been released. When can I download the source code on GitHub?

**Comment by primetomas** [comment]
Community v9.0 has not been released yet. It will be announced here.

**Reply by aloeenmae** [reply]
@primetomas is the 9.0 CE version planned to be released within this year or rather early next year?

**Reply by primetomas** [reply]
It's planned within this year.

**Reply by aloeenmae** [reply]
@primetomas any news about EJBCA-CE 9.x and release date ?;)

**Reply by primetomas** [reply]
It will be out before Christmas.
